[docs] update buildpacks tutorial to custom builder (#3166)
GoogleContainerTools/skaffold: docs/content/en/docs/tutorials/custom-builder.md

---

> title: "Building Artifacts with Custom Builder"

Reviewer: ```suggestion
title: "Building Artifacts with a Custom Build Script"
```

> linkTitle: "Custom Builder"

Reviewer: ```suggestion
linkTitle: "Custom Build Script"
```

weight: 100
---

> This page describes building Skaffold artifacts using custom builder [buildpacks](https://buildpacks.io/).

Reviewer: ```suggestion
This page describes building Skaffold artifacts using a custom build script, which builds images using [buildpacks](https://buildpacks.io/).
```

> Buildpacks enables building language-based containers from source code, without the need for a Dockerfile.

Reviewer: I think the way it was before was right

$ pack set-default-builder <insert buildpack image here>
```

## Tutorial - Hello World in Go

> To walk through a buildpacks custom builder tutorial, see our [buildpacks example](https://github.com/GoogleContainerTools/skaffold/tree/master/examples/buildpacks).

Reviewer: ```suggestion
This tutorial will be based on the [buildpacks example](https://github.com/GoogleContainerTools/skaffold/tree/master/examples/buildpacks) in our repository.
```

## Adding a Custom Builder to Your Skaffold Project

> Now, configure your Skaffold config to build artifacts with this custom builder.

Reviewer: ```suggestion
We'll need to configure your Skaffold config to build artifacts with this custom builder.
```
